src/app/features/home/post-capture-tab/series/series.page.html

```diff
@@ -0,0 +1,48 @@
+<mat-toolbar color="primary" *transloco="let t">
+  <button routerLink=".." routerDirection="back" mat-icon-button>
+    <mat-icon>arrow_back</mat-icon>
+  </button>
+  <div class="toolbar-spacer"></div>
+  <button
+    (click)="openContactSelectionDialog()"
```

Comment:
`openContactSelectionDialog()` does not exist at all. Make sure to remove unused method and code blocks before copy-pasting. There are other unused code blocks within this PR, you __MUST__ remove all of them to pass the review.